src/Microsoft.AspNet.Authentication.Cookies/CookieAuthenticationHandler.cs

```diff
@@ -279,6 +279,7 @@ private CookieOptions BuildCookieOptions()
             var context = new CookieSigningOutContext(
                 Context,
                 Options,
+                new AuthenticationProperties(signOutContext.Properties),
```

Comment:
CookieSigningInContext for example takes several properties in the ctor and has get/set for each, I would think its either or, public set (no ctor), or private set in ctor, with strong preference towards private/ctor IMO